src/csharp/Grpc.Tools/ProtoCompile.cs

```diff
@@ -123,6 +126,81 @@ public class ProtoCompile : ToolTask
                                                         "javanano", "js", "objc",
                                                         "php", "python", "ruby" };
 
+        static List<ErrorListFilter> ErrorListFilters = new List<ErrorListFilter>() 
+        {
+            // Example error with location
+            //../Protos/greet.proto(14) : error in column=10: "name" is already defined in "Greet.HelloRequest".
+            new ErrorListFilter
+            {
+                Pattern = new Regex(
+                    pattern: "(?'FILENAME'.+)\\((?'LINE'\\d+)\\) : error in column=(?'COLUMN'\\d+): (?'TEXT'.*)",
+                    options: RegexOptions.Compiled | RegexOptions.IgnoreCase,
+                    matchTimeout: TimeSpan.FromMilliseconds(100)),
```

Comment:
Move the timeout to a private static readonly TimeSpan field and share between filters.